amplify/team-provider-info.json

```diff
@@ -17,7 +17,8 @@
           "deploymentBucketName": "amplify-stockapp-dev-151619-deployment",
           "s3Key": "amplify-builds/expressServer-753174514c376f647362-build.zip",
           "jwtSecret": "$Ckpc6ZlGF*Z3vJ89Nr",
-          "environment": "prod"
+          "environment": "prod",
+          "fromEmail": "'caolandevelopment@gmail.com'"
```

Comment:
Need to maybe change this before deploying, easer to just to it now and finish it right.